lld/ELF/ScriptParser.cpp

```diff
@@ -235,6 +236,21 @@ void ScriptParser::readVersionScriptCommand() {
   }
 }
 
+void ScriptParser::readNoCrossRefs(bool to) {
+  expect("(");
+
+  script->nocrossrefs.push_back({});
+
+  script->nocrossrefs.back().firstOnly = to;
+
+  while (!atEOF() && !errorCount() && peek() != ")") {
+    StringRef section = next();
```

Comment:
GNU ld silently ignores unknown section names. Maybe this is because some sections may appear only depending on some build options and warning may abort build process with `--fatal-warnings` specified. 

Anyway, i don't have strong opinion here, I just think it might be a good idea to be fully compatible with GNU ld.

Reply:
For an initial version I agree that being compatible with GNU ld is worthwhile.